sovware/directorist
# Fix listing quick edit directory type select

## PR Type
What kind of change does this PR introduce?


## Description
How to reproduce the issue or how to test the changes

0. Make sure multi-directory is enabled
1. Go to directory listings view
2. Click on quick edit
3. See the directory type is not selected

## Any linked issues
Fixes #1346 

## Checklist

- [] My code follows the [WordPress coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)
